dbx/encrypt-fields.rst

```diff
@@ -11,58 +11,73 @@ In-Use Encryption
 Overview
 --------
 
-You can encrypt fields in a document using a set of features called
-**in-use encryption**.
-
-In-use encryption enables your client applications to encrypt data
-*before* sending it to MongoDB, and to query documents with encrypted fields.
+You can use the {+driver-short+} to encrypt document fields by using a
```

Comment:
Suggestion:

I think it can be helpful to identify that the user chooses the fields rather than them all being encrypted.

```suggestion
You can use the {+driver-short+} to encrypt specific document fields by using a
```